src/backend/index.ts

```diff
@@ -52,6 +53,7 @@ app.use('/change-requests', changeRequestsRouter);
 app.use('/description-bullets', descriptionBulletsRouter);
 app.use('/tasks', tasksRouter);
 app.use('/reimbursement-requests', reimbursementRequestsRouter);
+app.use('/materials', materialsController);
```

Comment:
this should be in the projects router, under the BOM section. The route on the left should match the one in the ticket description and the right should be projectsController.deleteMaterial (you should add the deleteMaterial endpoint on the projects stuff, rather than creating new files)